### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by sujitpatil29**
added

### CODE REVIEW

File: arguments.py [hunk 1]
```diff
@@ -11,4 +11,15 @@ def addition(a, b, c):
 res = addition(x, y, z)
 print(f"Addition of {x}, {y} and {z} is {res}")
 
+def sub(a, b, c):
+    ans = a - b - c
+    return ans
+
```
Comment (rushiwarekar) [inline arguments.py:17]: coommect pls

Comment (sujitpatil29) [reply arguments.py:17]: accepted

File: arguments.py [hunk 2]
```diff
@@ -11,4 +11,15 @@ def addition(a, b, c):
...
+x = 50
+y = 60
+z = 70
+
```
Comment (Gaurirohanpawar) [inline arguments.py:21]: for check

Comment (sujitpatil29) [reply arguments.py:21]: done

**Pull request merged**